client/blocks/inline-help/popover.jsx

```diff
@@ -208,54 +187,14 @@ class InlineHelpPopover extends Component {
 	};
 
 	renderPrimaryView = () => {
-		const { translate, siteId, showOptIn, showOptOut, isCheckout, isEditorDeprecated } = this.props;
+		const { siteId, isCheckout } = this.props;
 
 		// Don't show additional items inside Checkout.
 		if ( isCheckout ) {
 			return null;
 		}
 
-		return (
-			<>
-				<QueryActiveTheme siteId={ siteId } />
-				{ showOptOut && ! isEditorDeprecated && (
-					<Button
-						onClick={ this.switchToClassicEditor }
-						className="inline-help__classic-editor-toggle"
-					>
-						{ translate( 'Switch to Classic Editor' ) }
-					</Button>
-				) }
-
-				{ showOptIn && ! isEditorDeprecated && (
-					<Button
-						onClick={ this.switchToBlockEditor }
-						className="inline-help__gutenberg-editor-toggle"
-					>
-						{ translate( 'Switch to Block Editor' ) }
-					</Button>
-				) }
-			</>
-		);
-	};
-
-	switchToClassicEditor = () => {
-		const { siteId, onClose, optOut, classicUrl, translate } = this.props;
-
-		const proceed =
-			typeof window === 'undefined' ||
-			window.confirm( translate( 'Are you sure you wish to leave this page?' ) );
-
-		if ( proceed ) {
-			optOut( siteId, classicUrl );
-			onClose();
-		}
-	};
-
-	switchToBlockEditor = () => {
-		const { siteId, onClose, optIn, gutenbergUrl } = this.props;
-		optIn( siteId, gutenbergUrl );
-		onClose();
+		return <QueryActiveTheme siteId={ siteId } />;
```

Comment:
do you know why we need to keep querying by the active theme? I couldn't find any reference to it.

Reply:
I think `renderPrimaryView()` can be completely removed, since it isn't using any data about the theme. Before, it [was getting the active theme](https://github.com/Automattic/wp-calypso/blob/ff9fa608ca13949eb6147151bfb71dca64e0b977/client/blocks/inline-help/popover.jsx#L430) using the `getActiveTheme()` selector, but it has been removed so it doesn't make sense to keep querying it.
It will allow cleaning the code even more. For instance, `isCheckout` won't be used anymore either.

Reply:
I can't find any reference to it either. I was probably a little too focused on removing just editor opt-in/out and deprecation code. Thanks for the extra attention on this!